plugins/basic/python/tk_maya_basic/plugin_logging.py

```diff
@@ -41,15 +38,24 @@ def emit(self, record):
 
         :param record: Logging record to display in Maya script editor.
         """
-        # Give a standard format to the message.
-        msg = self.format(record)
 
-        # Display the message in Maya script editor in a thread safe manner.
+        # Give a standard format to the message:
+        #     Shotgun DEBUG: <message>
+        # or
+        #     Shotgun: <message>
+        # We use a simpler message format than the one used by the engine.
+        if record.levelno < logging.INFO:
+            msg = "Shotgun DEBUG: %s" % self.format(record)
```

Comment:
In maya, the output in the script editor looks like this:

```
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Bold.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-CondLight.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Italic.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Light.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Regular.ttf // 
// Shotgun: Toolkit platform was restarted. // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Bold.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-CondLight.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Italic.ttf // 
// Warning: Shotgun: Unable to load font file: /Users/manne/Library/Caches/Shotgun/wintermute/site.basic.maya/cfg/install/core/python/tank/platform/qt/fonts/OpenSans/OpenSans-Light.ttf // 
```

The debug information should be output consistently, so generate entries on this form:

```
// Debug: Shotgun: Toolkit platform was restarted. // 
```